debian/patches/series

```diff
@@ -0,0 +1 @@
+make-soname-be-the-major-version
```

Comment:
Usually we prefix the patch with `endless_` and standard is to attach a number to it (i.e. `10_some_patch`) so this should be something like `10_endless_atch_name`